### PR TITLE
feat: [DE] Create Pay in 30 view for universal modal

### PR DESCRIPTION
**Pull request by erikacolette29**

### CODE REVIEW

File: content/modals/DE/pay_in_thirty.json [hunk 1]
```diff
@@ -1,6 +1,37 @@
...
+            "formattedMaxAmount": "${CREDIT_OFFERS_DS.formattedMaxAmount}",
+            "formattedTotalCost": "${CREDIT_OFFERS_DS.formattedTotalCost}",
+            "minAmount": "${CREDIT_OFFERS_DS.minAmount}",
+            "maxAmount": "${CREDIT_OFFERS_DS.maxAmount}"
+        }
     },
-    "content": {}
+    "content": {
+        "headline": "Kaufen Sie jetzt was Sie möchten. Bezahlen Sie erst in 30 Tagen.",
+        "subheadline": "Erhältlich für Einkäufe i.H.v. 1€ - 1.000€.",
```
Comment (yeul) [inline content/modals/DE/pay_in_thirty.json]:
```suggestion
        "subheadline": "Erhältlich für Einkäufe i.H.v. {formattedMinAmount} - {formattedMaxAmount}.",
```

File: content/modals/DE/pay_in_thirty.json [hunk 2]
```diff
@@ -1,6 +1,37 @@
...
+        "instructions": [
+            "Wählen Sie PayPal als Bezahlmethode aus und nutzen Sie die Option,<strong> 30 Tage später zu bezahlen.</strong>",
+            "Schließen Sie Ihren Einkauf in wenigen Schritten ab.",
+            "Sie benötigen mehr als 30 Tage? Gegen eine Gebühr kann die Zahlung aufgeschoben werden."
+        ],
+        "disclosure": [
+            "Erhältlich für Einkäufe i.Hv. 1€ - 1.000€. Vorbehaltlich Kreditwürdigkeitsprüfung. Anspruchsberechtigte Kunden müssen PayPal ein SEPA-Lastschriftmandat erteilen sowie über ein deutsches PayPal-Privatkonto mit einem Bankkonto als Zahlungsquelle verfügen."
```
Comment (yeul) [inline content/modals/DE/pay_in_thirty.json]: Same as above, let's use our dynamic values for formattedMinAmount and formattedMaxAmount.

File: content/modals/US/product_list.json [hunk 3]
```diff
@@ -2,40 +2,5 @@
...
-        }
-    }
+    "content": {}
```
Comment (yeul) [inline content/modals/US/product_list.json]: Did you mean to undo this change?

File: content/modals/DE/product_list.json [hunk 4]
```diff
@@ -2,5 +2,31 @@
     "meta": {
```
Comment (yeul) [inline content/modals/DE/product_list.json]: Since this was just for testing I think we can clear this file back out.

File: content/modals/DE/pay_in_thirty.json [hunk 5]
```diff
@@ -1,6 +1,37 @@
...
+        ],
+        "navLinkPrefix": "Um Ihren Einkauf in monatlichen Raten zu zahlen,",
+        "linkToProductList": "wählen Sie die PayPal Ratenzahlung."
+    }
 }
```
Comment (yeul) [inline content/modals/DE/pay_in_thirty.json]: Let's rename this file `pay_in_1.json` to match our naming conventions of the product category.

File: src/components/modal/v2/parts/views/PayInThirty/Content.jsx [hunk 6]
```diff
@@ -0,0 +1,76 @@
+/* eslint-disable eslint-comments/disable-enable-pair */
```
Comment (yeul) [inline src/components/modal/v2/parts/views/PayInThirty/Content.jsx]: Let's rename the main view folder `PayIn1`.

File: src/components/modal/v2/parts/BodyContent.jsx [hunk 7]
```diff
@@ -72,7 +72,8 @@ const BodyContent = () => {
             <ShortTerm content={content} productMeta={productMeta} openProductList={openProductList} />
         ),
         PAYPAL_CREDIT_NO_INTEREST: <NI content={content} openProductList={openProductList} />,
-        PRODUCT_LIST: <ProductList content={content} setViewName={setViewName} />
+        PRODUCT_LIST: <ProductList content={content} setViewName={setViewName} />,
+        PAY_LATER_PAY_IN_1: <PayInThirty content={content} openProductList={openProductList} />
```
Comment (yeul) [inline src/components/modal/v2/parts/BodyContent.jsx]:
```suggestion
        PAY_LATER_PAY_IN_1: <PayIn1 content={content} openProductList={openProductList} />
```

File: src/components/modal/v2/parts/BodyContent.jsx [hunk 8]
```diff
@@ -11,7 +11,7 @@ import {
     useTransitionState
 } from '../lib';
 import Header from './Header';
-import { LongTerm, ShortTerm, NI, ProductList } from './views';
+import { LongTerm, ShortTerm, NI, ProductList, PayInThirty } from './views';
```
Comment (yeul) [inline src/components/modal/v2/parts/BodyContent.jsx]:
```suggestion
import { LongTerm, ShortTerm, NI, ProductList, PayIn1 } from './views';
```

File: src/components/modal/v2/parts/views/index.js [hunk 9]
```diff
@@ -2,3 +2,4 @@ export * from './LongTerm/Content';
 export * from './ProductList/Content';
 export * from './ShortTerm/Content';
 export * from './NI/Content';
+export * from './PayInThirty/Content';
```
Comment (yeul) [inline src/components/modal/v2/parts/views/index.js]:
```suggestion
export * from './PayIn1/Content';
```

File: content/modals/US/product_list.json (hunk 3 above)
Comment (erikacolette29) [reply content/modals/US/product_list.json]: yeah I grabbed that from what should be in merlins pr

File: utils/devServerProxy/config/DE/index.js [hunk 10]
```diff
@@ -17,7 +17,75 @@ export default {
...
+        modalViews: [
+            {
+                template: 'pay_in_1.json',
+                product: 'PAY_LATER_PAY_IN_1'
+            }
+        ],
+        messageThresholds: [
+            {
+                amount: 0.01,
+                template: 'pi30.json',
```
Comment (yeul) [inline utils/devServerProxy/config/DE/index.js]:
```suggestion
                template: 'pi30nq.json',
```

File: utils/devServerProxy/config/DE/index.js [hunk 11]
```diff
@@ -17,7 +17,75 @@ export default {
...
+                product: 'PAY_LATER_PAY_IN_1'
+            },
+            {
+                amount: 1,
+                template: 'pi30.json',
```
Comment (yeul) [inline utils/devServerProxy/config/DE/index.js]:
```suggestion
                template: 'pi30q.json',
```

File: utils/devServerProxy/config/DE/index.js [hunk 12]
```diff
@@ -17,7 +17,75 @@ export default {
...
+                product: 'PAY_LATER_PAY_IN_1'
+            },
+            {
+                amount: 1000.01,
+                template: 'pi30.json',
```
Comment (yeul) [inline utils/devServerProxy/config/DE/index.js]:
```suggestion
                template: 'pi30nq.json',
```

File: utils/devServerProxy/config/DE/index.js [hunk 13]
```diff
@@ -17,7 +17,75 @@ export default {
...
+                product: 'PAY_LATER_PAY_IN_1'
+            }
+        ],
+        offers: {
+            totalPayents: 1,
+            apr: 9.99,
+            nominalRate: 0,
+            minAmount: 1,
+            maxAmount: 1000
+        }
```
Comment (yeul) [inline utils/devServerProxy/config/DE/index.js]:
```suggestion
        offers: {
               PAY_LATER_PAY_IN_1: [
                {
                    totalPayents: 1,
                    apr: 9.99,
                    nominalRate: 0,
                    minAmount: 1,
                    maxAmount: 1000
                }
            ]
        }
```

File: tests/functional/v2/config/DE/DEV_DE_PAY_IN_1.js [hunk 14]
```diff
@@ -0,0 +1,33 @@
...
+    description: 'DE merchant eligible for pay in 1',
+    minAmount: 1,
+    maxAmount: 1000,
+    amounts: [
+        {
+            value: '0.01',
+            message: 'Non-qualifying Pay in 1',
+            expectedValue: 'Bezahlen Sie bei Einkäufen von 1€ bis 1.000€ erst nach 30 Tagen',
+            modalContent: {
+                subheadline: 'Erhältlich für Einkäufe i.H.v. 1€ -1.000€.'
```
Comment (yeul) [inline tests/functional/v2/config/DE/DEV_DE_PAY_IN_1.js]:
```suggestion
                subheadline: 'Erhältlich für Einkäufe i.H.v. 1€ - 1.000€.'
```

File: tests/functional/v2/config/DE/DEV_DE_PAY_IN_1.js [hunk 15]
```diff
@@ -0,0 +1,33 @@
...
+            }
+        },
+        {
+            value: '1.00',
+            message: 'Qualifying Pay in 1',
+            expectedValue: 'Bezahlen Sie nach 30 Tagen',
+            modalContent: {
+                subheadline: 'Erhältlich für Einkäufe i.H.v. 1€ -1.000€.'
```
Comment (yeul) [inline tests/functional/v2/config/DE/DEV_DE_PAY_IN_1.js]:
```suggestion
                subheadline: 'Erhältlich für Einkäufe i.H.v. 1€ - 1.000€.'
```

File: tests/functional/v2/config/DE/DEV_DE_PAY_IN_1.js [hunk 16]
```diff
@@ -0,0 +1,33 @@
...
+            }
+        },
+        {
+            value: '1000.01',
+            message: 'Non-qualifying Pay in 1',
+            expectedValue: 'Bezahlen Sie bei Einkäufen von 1€ bis 1.000€ erst nach 30 Tagen',
+            modalContent: {
+                subheadline: 'Erhältlich für Einkäufe i.H.v. 1€ -1.000€.'
```
Comment (yeul) [inline tests/functional/v2/config/DE/DEV_DE_PAY_IN_1.js]:
```suggestion
                subheadline: 'Erhältlich für Einkäufe i.H.v. 1€ - 1.000€.'
```

File: src/components/modal/v2/parts/views/PayIn1/Content.jsx [hunk 17]
```diff
@@ -0,0 +1,75 @@
...
+    // Optional outbound link to MPP product learn more page
+    const renderLearnMoreLink = () => {
+        return (
+            learnMoreLink && (
+                <div className="learnMoreLink__container">
+                    <InlineLinks text={learnMoreLink} />
+                </div>
+            )
+        );
+    };
```
Comment (yeul) [inline src/components/modal/v2/parts/views/PayIn1/Content.jsx]:
```suggestion
```
We can remove this.

File: src/components/modal/v2/parts/views/PayIn1/Content.jsx [hunk 18]
```diff
@@ -0,0 +1,75 @@
...
+                                    <div className="branded-image">
+                                        {/* TODO: include Icon component when desktop images are final */}
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                        <div className="content__row disclosure">
+                            <p>
+                                <InlineLinks text={currencyFormat(disclosure)} />
+                            </p>
```
Comment (yeul) [inline src/components/modal/v2/parts/views/PayIn1/Content.jsx]:
```suggestion
                            <p> {currencyFormat(disclosure)} </p>
```

File: src/components/modal/v2/parts/views/PayIn1/Content.jsx [hunk 19]
```diff
@@ -0,0 +1,75 @@
...
+                        </div>
+                        <div className="content__row productLink">
+                            {renderLearnMoreLink()}
```
Comment (yeul) [inline src/components/modal/v2/parts/views/PayIn1/Content.jsx]:
```suggestion
```

File: src/components/modal/v2/parts/views/PayIn1/Content.jsx [hunk 20]
```diff
@@ -0,0 +1,75 @@
...
+    return (
+        <Fragment>
+            <style>{styles._getCss()}</style>
+            <div className="content__container">
+                <main className="main">
+                    <div className="content__body">
+                        <div className="dynamic__container">
+                            <div className="content__row dynamic">
+                                <div className="content__col">
+                                    <div className="content__row donuts">
```
Comment (yeul) [inline src/components/modal/v2/parts/views/PayIn1/Content.jsx]:
```suggestion
                                    <div className="content__row instructions__container">
```

File: src/components/modal/v2/parts/views/PayIn1/styles.scss [hunk 21]
```diff
@@ -0,0 +1,161 @@
...
+        }
+
+        .content__row {
+
+            .content__col {
+                flex-wrap: wrap;
+                flex-direction: initial;
+            }
+
+            .donuts {
```
Comment (yeul) [inline src/components/modal/v2/parts/views/PayIn1/styles.scss]:
```suggestion
            .instructions__container {
```

File: src/components/modal/v2/parts/views/PayIn1/styles.scss [hunk 22]
```diff
@@ -0,0 +1,161 @@
...
+                }
+
+                .branded-image {
+                    display: flex;
+                    margin-top: 56px;
+                }
+            }
+
+            &.instructions {
+                padding: 0 25px;
```
Comment (yeul) [inline src/components/modal/v2/parts/views/PayIn1/styles.scss:100]:
```suggestion
                padding: 0 15px;
```

File: src/components/modal/v2/parts/views/PayIn1/styles.scss [hunk 23]
```diff
@@ -0,0 +1,161 @@
...
+                    }
+
+                    li {
+                        font-size: 1.125rem;
+                        line-height: 24px;
+                    }
+                }
+            }
+
+            &.disclosure {
```
Comment (yeul) [inline src/components/modal/v2/parts/views/PayIn1/styles.scss:130]:
```suggestion
            &.disclosure {
            margin-top: 20px;
            
```

File: src/components/modal/v2/parts/views/PayIn1/styles.scss [hunk 24]
```diff
@@ -0,0 +1,161 @@
...
+                    @include tablet {
+                        margin: 1em auto;
+                        max-width: 315px;
+                    }
+
+                    @include mobile {
+                        max-width: none;
+                    }
+                }
+            }
```
Comment (yeul) [inline src/components/modal/v2/parts/views/PayIn1/styles.scss]:
```suggestion
```
We can remove this.

File: src/components/modal/v2/parts/views/PayIn1/styles.scss (hunk 23 above)
Comment (erikacolette29) [reply src/components/modal/v2/parts/views/PayIn1/styles.scss:130]: is this to replace padding-top?

Comment (yeul) [reply src/components/modal/v2/parts/views/PayIn1/styles.scss:130]: yeah we can just push the disclosure a little bit further down to give it a bit more breathing room

File: src/components/modal/v2/parts/views/PayIn1/Content.jsx [hunk 25]
```diff
@@ -0,0 +1,60 @@
...
+    return (
+        <Fragment>
+            <style>{styles._getCss()}</style>
+            <div className="content__container">
+                <main className="main">
+                    <div className="content__body">
+                        <div className="dynamic__container">
+                            <div className="content__row dynamic">
+                                <div className="content__col">
+                                    <div className="content__row instructions_container">
```
Comment (yeul) [inline src/components/modal/v2/parts/views/PayIn1/Content.jsx]:
```suggestion
                                    <div className="content__row instructions__container">
```

File: src/components/modal/v2/parts/views/PayIn1/styles.scss [hunk 26]
```diff
@@ -0,0 +1,139 @@
...
+            margin-top: 0px;
+        }
+
+        .content__row {
+            .content__col {
+                flex-wrap: wrap;
+                flex-direction: initial;
+            }
+
+            .instructions__container {
```
Comment (yeul) [inline src/components/modal/v2/parts/views/PayIn1/styles.scss]:
```suggestion
            &.instructions__container {
```

File: src/components/modal/v2/parts/views/PayIn1/styles.scss [hunk 27]
```diff
@@ -0,0 +1,139 @@
+@import '../../../styles/globals/colors';
+@import '../../../styles/globals/mixins';
+
```
Comment (yeul) [inline src/components/modal/v2/parts/views/PayIn1/styles.scss:3]:
```suggestion

h1 {
    @include tablet {
        max-width: 315px;
    }
}

```
